docs/content/guides/6.multistore/2.tooling-and-concepts/4.deployment/7.configuration.md

```diff
@@ -7,11 +7,226 @@ navigation:
+::
+
+## Configuration Parameters
+
+### `projectName`
+
+The `projectName` parameter is a crucial identifier that links your store to Alokai Console.
+
+::tip Finding Your Project Name
+You can find your project name in the [Deployment Variables](/console/instance/settings/deployment-variables) section of the Alokai Console.
+::
+
+Example configuration:
+```json
+{
+  "stores": {
+    "fashion-brand": {
+      "deployment": {
+        "projectName": "fashion-brand-prod",  // Must match Alokai Console project name
+        // ... other parameters
+      }
+    }
+  }
+}
+```
+
+### `region`
+
+The `region` parameter determines where your store will be deployed. 
+
+::warning `region` Required
+Deployment cannot be triggered without a valid region configuration. The `region` value can be found in the [Deployment Variables](/console/instance/settings/deployment-variables) section of the Alokai Console. Ensure the value in `alokai.config.json` matches the region selected for your project.
+::
+
+Example region configuration:
+```json
+{
+  "stores": {
+    "fashion-brand": {
+      "deployment": {
+        "region": "us-east-1",  // AWS region identifier
+        // ... other parameters
+      }
+    }
+  }
+}
+```
+
+### `framework`
+
+::info Auto-detection
+The `framework` parameter is optional when your project uses only one frontend framework (Next.js or Nuxt). In this case, Alokai CLI will automatically detect and use the correct framework.
+::
+
+The `framework` parameter defines whether a store uses Next.js or Nuxt. Alokai supports both frameworks out of the box through two base applications:
+- `apps/storefront-unified-nextjs` for Next.js stores
+- `apps/storefront-unified-nuxt` for Nuxt stores
+
+Each store in `alokai.config.json` may specify which framework it will use for deployment:
+
+```json
+{
+  "stores": {
+    "fashion-brand": {
+      "deployment": {
+        "framework": "nextjs",  // This store will deploy using Next.js
+        // ... other parameters
+      }
+    },
+    "sports-brand": {
+      "deployment": {
+        "framework": "nuxt",  // This store will deploy using Nuxt
+        // ... other parameters
+      }
+    }
+  }
+}
+```
+
+Your project structure can support both frameworks simultaneously. For example:
+```bash
+apps/
+├── storefront-unified-nextjs/     # Base Next.js implementation
+├── storefront-unified-nuxt/       # Base Nuxt implementation
+└── stores/
+    ├── fashion-brand/            
+    │   ├── storefront-unified-nextjs/  # Next.js Storefront
+    └── sports-brand/
+        └── storefront-unified-nuxt/    # Nuxt Storefront
+```
+
+This dual-framework support is particularly beneficial when:
+- Different teams specialize in different frameworks
+- You're migrating from one framework to another
+- Specific stores have requirements better suited to a particular framework
+
+## Local Test Deployments
```

Comment:
The name "Local Test Deployment" is misleading. My first impression was that we deploy it to local docker not the cloud. What it does in fact is: build a storefront locally and deploy from the command line.
Maybe we could call it: "Local build & deployment via CLI"

Reply:
Updated https://github.com/vuestorefront/vue-storefront/pull/7398/commits/27ded587eb3ffb0c55826066812100243916b669